Resources/doc/1-setting_up_the_bundle.md

```diff
@@ -12,30 +24,28 @@ Step 1: Setting up the bundle
 > But in this case, you need to manually set it up and configure FOSRestBundle to use it
 > via the ``service`` section in the app config
 
-Simply run assuming you have installed composer.phar or composer binary:
-
-``` bash
-$ php composer.phar require friendsofsymfony/rest-bundle 1.3.*
-```
-
-### B) Enable the bundle
+B: Enable the Bundle
+--------------------
 
-Finally, enable the bundle in the kernel:
+Then, enable the bundle by adding the following line in the `app/AppKernel.php`
+file of your project:
 
-``` php
+```php
 <?php
 // app/AppKernel.php
 
-public function registerBundles()
+// ...
+class AppKernel extends Kernel
 {
-    $bundles = array(
+    public function registerBundles()
+    {
+        $bundles = array(
+            // ...
+            new FOS\RestBundle\FOSRestBundle(),
+        );
+    
         // ...
-        new FOS\RestBundle\FOSRestBundle(),
-        
-        // if you choose to use JMSSerializer, make sure that it is registered in your application
-        
-        // new JMS\SerializerBundle\JMSSerializerBundle(),
```

Comment:
in the note about JMSerializer, it states that the user should follow its own installation instructions to set-up and install it. I don't think this bundle should cover that.